src/Utilities.jl

```diff
@@ -100,33 +101,50 @@ Create output directories for the experiment. If `comms_ctx` is provided, only t
 By default, the regrid directory is created as a temporary directory inside the output directory,
 and the artifacts directory is created inside the output directory with the name `artifacts/`.
 
+`ClimaUtilities.OutputPathGenerator` is used so that simulations can be re-run and re-started.
+The output path looks like:
+```
+coupler_output_dir_amip/
+├── checkpoints
+│       └── checkpoints for the various models
+├── output_0000/
+│   ├── atmos/
+│   │   └── output of the atmos model
+│   └── ocean/
+│       └── output of the ocean model
+├── output_0001/
+│   └── ... component model outputs in their folders ...
+├── output_0002/
+│   └── ... component model outputs in their folders ...
+└── output_active -> output_0002/
+```
+
 # Arguments
 - `output_dir::String`: The directory where the output files will be stored. Default is the current directory.
 - `regrid_dir::String`: The directory where the regridded files will be stored. Default is `output_dir/regrid_tmp/`.
-- `artifacts_dir::String`: The directory where the artifacts will be stored. Default is `output_dir/artifacts/`.
+- `checkpoint_dir::String`: The directory where the checkpoint files will be stored. Default is `output_dir/checkpoints/`.
+- `artifacts_dir::String`: The directory where the artifacts will be stored. Default is `output_dir_artifacts/`.
```

Comment:
```suggestion
- `artifacts_dir::String`: The directory where the artifacts will be stored. Default is `output_dir/artifacts/`.
```

Reply:
I think we should get rid of this at some point (because artifacts is relevant only for buildkite). In the meantime, I added to the diagram